src/libraries/EventsLib.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title EventsLib
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Library exposing events.
+library EventsLib {
+    /// @notice Emitted when the merkle tree's root is updated.
+    /// @param newRoot The new merkle tree's root.
+    /// @param newIpfsHash The optional ipfs hash containing metadata about the root (e.g. the merkle tree itself).
+    event RootUpdated(bytes32 indexed newRoot, bytes32 indexed newIpfsHash);
```

Comment:
We use past tense instead of function names here?

Reply:
I find it more natural tbh

Reply:
Right, non blocking to me

Reply:
We could harmonize with blue but not a blocker as well